integrity-backend/input-chatbot-examples/generic-meta-content.json

```diff
@@ -0,0 +1,28 @@
+{
+  "contentMetadata":
+  {
+    "name": "Chat archive bot",
+    "mime": "application/zip",
+    "description": "Chat archive bot",
+    "author":
+    {
+      "@type": "Organization",
+      "identifier": "https://starlinglab.org",
+      "name": "Starling Lab"
+    },
+    "dateCreated": "2022-03-29T13:54:20Z",
+    "extras":
+    {
+      "channels": ["channel1","channel2"],
+      "dateRage": {
+          "from": "2022-03-29T13:00:00Z",
+          "to": "2022-03-29T14:00:00Z"
+      }
+    },
+    "private": {
+        "phonenumber": "000-000-0000"
+        "account":"@telegramuser"
```

Comment:
Telegram use a bot so its not married to a phone number, the only phone number that i think we will be married to will be signal.

So maybe account, and maybe worksapce for slack?

these will have to be manual somewhere because i dont think they are extractable

Reply:
How is this in `private`?

```
      "signal":
      {
        "phoneNumber": "000-000-0000"
      },
      "slack":
      {
        "botAccount": "slackArchiveBot",
        "workspace": "slackWorkSpace"
      },
      "telegram":
      {
        "botAccount": "telegramArchiveBot"
      }
```